godel_msgs/srv/KeyenceProcessExecution.srv

```diff
@@ -0,0 +1,18 @@
+# Joint trajectory
+trajectory_msgs/JointTrajectory trajectory_approach
+trajectory_msgs/JointTrajectory trajectory_process
+trajectory_msgs/JointTrajectory trajectory_depart
+
+# wait for execution to finish before returning
+bool wait_for_execution
+
+# Use the simulated robot or the actual robot
+bool simulate
+
+# Parameters associated with the Keyence Process
+godel_msgs/ScanPlanParameters params
+
+---
+
+# Return code
+int32 code
```

Comment:
The naming of this service (and `BlendProcessExecution` and perhaps `KeyenceProcessPlanning`) seems odd: most services have names that describe an action (ie: `GetX` or `MoveToY`). From the comments it does seem like invoking these services will make the system as a whole do something, while the current name (to me, non-native speaker) seems more like a description of a particular phase in the entire process (there is planning, then execution, etc).

Should this be renamed to `ExecuteBlendingProcess` perhaps (and similar names for the others)?